server/src/main/java/io/spine/server/command/model/CommandReactionSignature.java

```diff
@@ -122,7 +121,7 @@ public boolean matches(MethodParams params) {
             @Override
             public boolean matches(MethodParams params) {
                 return params.are(EventMessage.class, EventContext.class)
-                        && params.firstIs(GeneratedMessageV3.class);
+                        && params.declaredAsClasses();
```

Comment:
Can we check that all params are declared as classes in a single place? Now, we have to copy this check for each signature.